scripts/initialize/main.go

```diff
@@ -77,13 +77,29 @@ func addResourcePermissions(db *bun.DB, basePath string) error {
 			if err != nil {
 				log.Fatal(err)
 			}
-			items = append(items, data)
+			existing := &models.ResourcePermission{}
+			err = db.NewSelect().Model(existing).Where("name = ?", data.Name).Scan(context.Background())
+			if err != nil && err != sql.ErrNoRows {
+				log.Fatal("Error verifying existing resource permissions ", err)
+			}
+			if err == sql.ErrNoRows {
+				_, err = dao.Create(context.Background(), db, &data)
+				if err != nil {
+					log.Fatal("Error inserting resource permissions ", err)
+				}
+
+			} else {
+				_, err = db.NewUpdate().Model(&data).Where("name = ?", data.Name).Exec(context.Background())
+				if err != nil {
+					log.Fatal("Error updating resource permissions ", err)
+				}
+
+			}
+
 		}
 	}
 
-	fmt.Println("Adding", len(items), "resource permissions")
-	_, err = dao.Create(context.Background(), db, &items)
-	return err
+	return nil
```

Comment:
we should return err in case of valid errors - we do not want to proceed with paralus without these necessary permissions.